src/components/Header/index.tsx

```diff
@@ -42,6 +42,9 @@ const Header = ({ t }: { t: TFunction }) => {
         <CustomNavLinkSmall onClick={() => scrollTo("product")}>

```

Comment:
- **Runtime error check**  
  The `scrollTo` function is called without checking if it exists. This could lead to a runtime error if `scrollTo` is null or undefined.